projects/core/components/svg/svg-options.ts

```diff
@@ -49,6 +56,23 @@ export const TUI_SVG_OPTIONS = new InjectionToken<TuiSvgOptions>(`[TUI_SVG_OPTIO
     }),
 });
 
+export const TUI_SVG_MULTI_SRC_PROCESSOR = new InjectionToken<
+    TuiHandler<TuiSafeHtml, TuiSafeHtml>
+>(`[TUI_SVG_MULTI_SRC_PROCESSOR]`);
+
+export function tuiSvgMultiSrcProcessor(
+    handler: TuiHandler<TuiSafeHtml, TuiSafeHtml>,
+): Provider {
+    return {
+        multi: true,
+        provide: TUI_SVG_MULTI_SRC_PROCESSOR,
+        useFactory: (
+            parentHandlers: Array<TuiHandler<TuiSafeHtml, TuiSafeHtml>> | null,
+        ) => [...(parentHandlers ?? []), handler],
+        deps: [[new SkipSelf(), new Optional(), TUI_SVG_MULTI_SRC_PROCESSOR]],
```

Comment:
nit: let's move deps above, because they kind of describe what's going in the factory.

Reply:
fixed